src/backend/api/reviews.js

```diff
@@ -0,0 +1,96 @@
+const { query } = require("express");
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const reviewList = await knex('review').select('*')
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
```

Comment:
I would expect an empty list if there are no reviews, so a 404 is not needed here.

Reply:
yaah..you are right. I have updated code